gradlew

```diff
@@ -78,14 +78,14 @@ if [ -n "$JAVA_HOME" ] ; then
         die "ERROR: JAVA_HOME is set to an invalid directory: $JAVA_HOME
 
 Please set the JAVA_HOME variable in your environment to match the
-location of your Java installation."
+position of your Java installation."
```

Comment:
...

Reply:
That's REALLY weird. I renamed the class, but I don't know why it changed the build scripts. I'll fix that

Reply:
Fixed